src/components/proposal_list/index.tsx

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import classnames from 'classnames';
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import { ProposalListProps } from './types';
+import { useGetStyles } from './styles';
+import {
+  Title, Button, Status,
+} from './component';
+
+const Component = (props: { current: boolean, display: string }) => {
+  if (props.current === true) {
+    return (
+      <div className={classnames('button')}>
+        <Button display={props.display} />
+      </div>
+    );
+  }
+  return (
+    <div className={classnames('status')}>
+      <Status display={props.display} />
+    </div>
+  );
+};
```

Comment:
instead of making a component in 

```
<Component {...row.status} />
```

we could have just

```
{row.status.current ? <Button display={row.status.display} /> : <Status display={row.status.display />}
```

what do you think?